packages/quantic/force-app/main/default/lwc/quanticResultsPerPage/e2e/fixture.ts

```diff
@@ -0,0 +1,74 @@
+import {ResultsPerPageObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const resultsPerPageUrl = 's/quantic-results-per-page';
+
+interface ResultsPerPageOptions {
+  initialChoice: number;
+  choicesDisplayed: string;
+}
+
+type QuanticResultsPerPageE2EFixtures = {
+  resultsPerPage: ResultsPerPageObject;
+  search: SearchObject;
+  options: Partial<ResultsPerPageOptions>;
+};
+
+type QuanticResultsPerPageE2ESearchFixtures =
+  QuanticResultsPerPageE2EFixtures & {
+    urlHash: string;
+  };
+
+type QuanticResultsPerPageE2eInsightFixtures =
+  QuanticResultsPerPageE2ESearchFixtures & {
```

Comment:
we should use the core type here and not the search one as the insight type should not contain `urlHash `
```suggestion
  QuanticResultsPerPageE2EFixtures & {
```

Reply:
It's like that everywhere though xD